models/DeliveryExecutionFinderService.php

```diff
@@ -101,9 +109,11 @@ protected function getBooleanOption(LtiLaunchData $launchData, string $option):
             : false;
 
         if (is_numeric($value)) {
-            $value = (bool)intval($value);
-        } else if ($value == 'true') {
+            $value = (bool)(int)$value;
```

Comment:
how about `$value = filter_var($value, FILTER_VALIDATE_BOOLEAN)` instead of all this if..else stuff?

Reply:
Fixed